app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+# class User < ApplicationRecord
+#   has_many :posts
+#   has_many :likes
+#   has_many :comments
+# end
+
+class User < ApplicationRecord
```

Comment:
Has User, Post, Comment and Like models inside app/models/ directory 🔥 Well done